### PR TITLE
update tox to test against hg 3.2.1

### PR DESCRIPTION
**Pull request by Callek**
@hwine r?

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -28,10 +28,10 @@ deps =
     {[base]deps}
     mercurial==2.6.2
 
-[testenv:py27-hg3.1]
+[testenv:py27-hg3.2]
 deps =
     {[base]deps}
-    mercurial==3.1.2
+    mercurial==3.2.1
```
Comment (hwine) [inline tox.ini]: I'd recommend also running tests against tip, or last release, of mercurial. E.g. we're running 3.3.3 on the server now.

Comment (Callek) [inline tox.ini]: These are client tests, and I'd rather not make our build farm checkins fail due to a "tip" version of hg. That said we probably should run them in a "fails ok" way.

Comment (hwine) [inline tox.ini]: wfm - r+

**Comment by Callek** [comment]
https://hg.mozilla.org/build/tools/rev/145730c24ed9

**Pull request closed without merging**